### DISCUSSION TITLE
being rate limited

### DISCUSSION THREAD

**Discussion by drewold123** [Vedza/NitroSniperGo#108] [Q&A] [answered]
why does it always say im being rate limited

**Comment by Vedza** [comment, marked as answer]
Because the sniper tried to claim to many gift links so Discord did that

**Reply by drewold123** [reply]
![image](https://user-images.githubusercontent.com/64940789/104325745-f704cc80-54b6-11eb-956c-f954198a96c4.png)

is this an issue on v2 windows build

**Reply by Vedza** [reply]
No it's not, it was also here before but hidden. I reformatted the code a bit and forgot that error, but the sniper will still work fine.

**Reply by Vedza** [reply]
Actually if you or someone else is available, I need someone to do some quick tests to hide that error. I cannot test it myself as I don't have this error with my token. Send me your discord here if you want to help me with that issue.

**Reply by Asvarius** [reply]
ay @Vedza I've found the issue, there are codes that are from Xbox game pass plan, so if you already activated a discord game pass code you can't activate it anymore
(not related to this thread)

**Reply by Vedza** [reply]
Oh ok thanks, I'm going to add that in the know issues. btw I think you can still answer to your issue even if it's closed
